### DISCUSSION TITLE
I need help

### DISCUSSION THREAD

**Discussion by colebolebole** [valinet/ExplorerPatcher#756] [Q&A]
So my taskbar looks like this ![Screenshot 1_30_2022 3_40_40 PM.png](https://user-images.githubusercontent.com/88512222/151717436-5a1e8ae7-4e76-450f-82b1-f364a8339896.png) and I want to change it to where there isn't text, I just want the default, anyone know how to change it?

**Comment by valinet** [comment]
https://github.com/valinet/ExplorerPatcher/wiki/All-features#combine-taskbar-icons-on-primarysecondary-taskbars

**Reply by colebolebole** [reply]
Thank you!
